_data/tagging-status.yml

```diff
@@ -1001,11 +1001,13 @@
 
  - name: csvsimple
    type: package
-   status: unknown
+   status: compatible
+   comments: "`\\csvautotabularray` etc. are not supported since tabularray is
```

Comment:
given the comment (and not knowing how important the command (and the etc) is for using the package, I wonder if it sholdb't be better classified as partially-compatible